### PR TITLE
Add progressive_rollout boolean to a Note

### LINKED ISSUE #810: Add a rollout indicator for release notes with a feature rollout
There is an increasing amount of requests for release notes about features shipped with a progressive rollout.
Currently, in this scenario, we start the release note with wording such as "Gradually rolling out in Fx119..."
To help users understand why they may not see a new feature even when they have updated to the latest release version,
This wording becomes repetitive as there can be multiple feature rollouts in the same release. 

As a Release Manager, I would like to set a flag in Nucleus that a release note is for gradual rollout, so that an indicator is displayed on the mozilla.org release notes page beside the note.

### PR DESCRIPTION
**Pull request by stevejalim**
This is the Nucleus-side change to support marking a release note as being on a progressive roll out.

I think this is all that's needed - the `SaveToGithub` base will take and save it to `mozilla/release-notes`, which in turn will get synced in to Bedrock to be shown as a badge/panel: https://github.com/mozilla/bedrock/pull/13894

Resolves #810

### CODE REVIEW

File: nucleus/rna/admin.py [hunk 1]
```diff
@@ -6,7 +6,6 @@
 from django.contrib import admin
 from django.utils.html import format_html
 from django.utils.timezone import now
-
```
Comment (pmac) [inline nucleus/rna/admin.py:9]: our standard is to keep django and 3rd party imports separated by a blank line. Any reason this one was removed?

**Comment by pmac** [comment]
We're not actually using the `SaveToGithub` stuff. How it works is there's an [action in the release-notes repo](https://github.com/mozilla/release-notes/actions/workflows/process-notes.yml) that fetches fresh release data via the https://nucleus.mozilla.org/rna/all-releases.json URL, then commits any changes to the repo. So in order for this new flag to make it to the repo you'll need to make sure the `Note.to_dict()` method outputs the new field. It looks like it should, but we should make sure.

File: nucleus/rna/admin.py (hunk 1 above)
Comment (stevejalim) [reply nucleus/rna/admin.py:9]: Wasn't deliberate - flake8/black I guess. I can reinstate

**Comment by stevejalim** [comment]
@pmac  yeah it's in the `/rna/all-releases.json` 

```
[
    {
        "product": "Firefox",
        "channel": "Release",
        "version": "200.0.0",
        "release_date": "2023-11-16",
        "text": "test release",
        "is_public": false,
        "bug_list": "bug list here",
        "bug_search_url": "",
        "system_requirements": "Lorem do, anim cillum laboris. Cillum laboris duis, est incididunt. Est incididunt esse cillum incididunt eu eu adipiscing. Esse, cillum incididunt eu eu. Eu eu adipiscing est.",
        "title": "Firefox 200.0.0 Release",
        "slug": "firefox-200.0.0-release",
        "created": "2023-11-16T14:36:37.663696+00:00",
        "modified": "2023-11-16T14:36:37.662330+00:00",
        "notes": [
            {
                "id": 1,
                "bug": null,
                "note": "Note note note, is a progressive rollout",
                "tag": "",
                "sort_num": 0,
                "is_public": true,
                "progressive_rollout": true,
                "created": "2023-11-16T14:37:29.502328+00:00",
                "modified": "2023-11-16T14:37:29.501049+00:00"
            },
            {
                "id": 2,
                "bug": null,
                "note": "note note Is not a progressive rollout",
                "tag": "",
                "sort_num": 0,
                "is_public": true,
                "progressive_rollout": false,
                "created": "2023-11-16T14:38:32.583167+00:00",
                "modified": "2023-11-16T14:38:32.582173+00:00"
            }
        ]
    }
]
```

**Review by pmac — APPROVED** [review]
🗒️

**Review by robhudson — APPROVED** [review]
🧨

**Pull request merged**